### ISSUE TITLE
[Feature request] Add tooltips to side menu

### ISSUE THREAD

**Issue by chaitanya-deep** [IFCjs/web-ifc-viewer#128]
Hi,

I just tried out the web-ifc-viewer (https://ifcjs.github.io/web-ifc-viewer/example/index)

I don't know what the second icon on the side menu means or does. I thought it's a download button but clicking on it does nothing.
I had to inspect it and see the filename as "section-plane-down.svg" and I think it repositions the model to ground plane?

**Comment by agviegas** [comment]
Hey! As I commented in other issue that you did, the main goal of the example is to provide a "hello-world" sample of how to use the library. We want to make something similar to the [three.js editor](https://threejs.org/editor/) to show all capabilities of the library in one viewer 🙂 Maybe that would be the place to implement a good GUI?

**Comment by chaitanya-deep** [comment]
Yes. That sounds like a place to implement a better GUI. Is there already a repo for the work?
